src/views/home/components/transaction-form/transaction-form.view.tsx

```diff
@@ -175,10 +192,21 @@ const TransactionForm: FC<TransactionFormProps> = ({
           </div>
         </div>
         <div className={`${classes.row} ${classes.middleRow}`}>
-          <div className={classes.tokenSelector}>
+          <button
+            className={classes.tokenSelector}
+            onClick={() =>
+              setList({
+                type: "token",
+                items: Object.values(env.tokens),
+                onClick: onTokenButtonClick,
+              })
+            }
+            type="button"
+          >
             <Icon url={token.logoURI} size={24} />
```

Comment:
We could consider using blending to better integrate the icons that lack alpha:

![icons-blending](https://user-images.githubusercontent.com/15896177/168655448-3590dd58-3b7f-4cff-905a-975c671b3f95.gif)

Reply:
That breaks other icons with white background
![Screenshot_20220517_184746](https://user-images.githubusercontent.com/19190248/168866202-63c735d6-e07f-4242-890c-522e14b88d78.png)

Reply:
Sure, it affects them all. Any other idea to make this looks better?